packages/b/box2d/xmake.lua

```diff
@@ -0,0 +1,125 @@
+package("box2d")
+
+    set_homepage("https://box2d.org")
+    set_description("A 2D Physics Engine for Games")
+
+    if is_plat("windows", "linux", "macosx") then
+        set_urls("https://github.com/erincatto/box2d/archive/v$(version).zip")
+        add_versions("2.4.0", "6aebbc54c93e367c97e382a57ba12546731dcde51526964c2ab97dec2050f8b9")
+    end
+
+    add_deps("cmake")
+
+    on_load(function (package)
+        if is_plat("windows") and package:config("build_testbed") then
+            package:add("syslinks", "User32", "Gdi32", "Shell32")
+        end
+    end)
+
+    add_configs("build_unit_tests", {description = "Build the Box2D unit tests", default = true, type = "boolean"})
+    add_configs("build_testbed",    {description = "Build the Box2D testbed", default = false, type = "boolean"})
+    add_configs("build_docs",       {description = "Build the Box2D documentation", default = false, type = "boolean"})
+
+    on_install("windows", "linux", "macosx", function (package)
+        local config = {}
+        local function add_config_arg(config_name, cmake_name)
+            table.insert(config, "-D" .. cmake_name .. "=" .. (package:config(config_name) and "ON" or "OFF"))
+        end
+
+        add_config_arg("build_unit_tests", "BOX2D_BUILD_UNIT_TESTS")
+        add_config_arg("build_testbed",    "BOX2D_BUILD_TESTBED")
+        add_config_arg("build_docs",       "BOX2D_BUILD_DOCS")
```

Comment:
I don't think there is a need to build documentation and tests, they will never be used. Because these packages are not installed in the system environment, they are only used to integrate xmake projects

Reply:
I thought I would just add these configs since these are options of cmake. But I could just set those to false on default. ```build_docs``` is already false on default. Or should I remove them completely?

Reply:
I think these configurations are not needed, because these configuration options are never enabled in add_requires. 

Please remove these unnecessary options and try to keep `package()` simple.